mmdet/models/dense_heads/base_dense_head.py

```diff
@@ -14,6 +14,15 @@ class BaseDenseHead(BaseModule, metaclass=ABCMeta):
     def __init__(self, init_cfg=None):
         super(BaseDenseHead, self).__init__(init_cfg)
 
+    def init_weights(self):
+        super(BaseDenseHead, self).init_weights()
+        # avoid init_cfg overwrite the initialization of `conv_offset`
+        for m in self.modules():
+            # DeformConv2dPack, ModulatedDeformConv2dPack
+            if hasattr(m, 'conv_offset'):
```

Comment:
DCN should not be able to use `init_cfg`.

Reply:
conv_offset is a Conv2D

Reply:
You can reproduce this bug by running NASFCOS